aiida/tools/query/calculation.py

```diff
@@ -125,6 +125,7 @@ def get_query_set(self, relationships=None, filters=None, order_by=None, past_da
             for projection in self._valid_projections
             if projection not in self._compound_projections
         ]
+        unique_projections = list(set(projected_attributes))
```

Comment:
Does this possibly change the order of projections? (Not sure if that makes a difference anywhere)

Reply:
Good catch, it _does_ change order. But it just changes the order of the projections in the query set. This is not what determines the final output order, which is done in `CalculationQueryBuilder.get_projected` which uses the original `projections` list.